source/_integrations/zha.markdown

```diff
@@ -402,6 +402,16 @@ Note that not all devices support binding as it depends on the Zigbee implementa
 
 Binding a remote directly to a bulb or group has the benefit of faster response time and smoother control. This greatly improves user feedback experience functions like dimming as the remote then directly dims the lightbulb and thus does not have to make the software roundtrip via the ZHA coordinator.
 
+## Zigbee backup and restore in ZHA
+
+Zigbee Home Automation (ZHA) integration now features Zigbee network backup, restore/recovery, and migrating between Zigbee coordinators since Home Assistant 2022.9 release. Backups are taken automatically however a single backup to file for easy download can also be manually created from the configuration page under Network Settings.
```

Comment:
- Our documentation reflects/matches the latest Home Assistant release, and isn't a changelog. Remove the version in the suggestion below for that reason.

```suggestion
Zigbee Home Automation (ZHA) integration now features Zigbee network backup, restore/recovery, and migrating between Zigbee coordinators. Backups are taken automatically however, a single backup to a file for easy download can also be manually created from the configuration page under Network Settings.
```